pweave/processors/base.py

```diff
@@ -32,10 +35,10 @@ def run(self):
         if self.documentationmode:
             success = self._getoldresults()
             if success:
-                print("Restoring cached results")
```

Comment:
@mpastell, it seems like you might already have two level of priority messages, one where you write to stdout as here, and one where you write to stderr as below. 

Would it make sense to have these be different log levels, particularly `warn` and `info`? By default, they would both go to stderr, but I'm not sure if that's a problem. 

Is there some reason why you wanted some messages to go to stdout and others to stderr?

Reply:
@piccolbo if you are taking on some commit responsibilities, would you mind taking a look at this question? Once this is resolved, I'll be happy to write some tests.

Reply:
I agree with your point. I think it's safe to assume this is better implemented as a logging statement according to unix tool convention (see https://monkey.org/~marius/unix-tools-hints.html for example and the Kernighan Pike book for the long version) and goes to stderr by default